docs/deployments/oc10-app.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Deploy as an app in ownCloud 10"
+date: 2018-05-02T00:00:00+00:00
+weight: 1
+geekdocRepo: https://github.com/owncloud/phoenix
+geekdocEditPath: edit/master/docs/deployments
+geekdocFilePath: oc10-app.md
+---
+
+{{< toc >}}
+
+One of the build artifacts of the ownCloud Web is a ownCloud 10 app bundle.
+
+## Prerequisites
+- Running [ownCloud 10 server](https://owncloud.com/download-server/) with version 10.6
+- Installed [oauth2 app](https://marketplace.owncloud.com/apps/oauth2)
+
+## Deploying ownCloud Web
+To download the ownCloud Web, go to https://github.com/owncloud/phoenix/releases and pick your desired version. In the release, you will find a file called `phoenix-app.tar.gz`.
+Download this file and move it into the `apps` folder in your ownCloud 10 server.
+
+{{< hint info >}}
+The app bundle is provided only from version 1.0.0
```

Comment:
If we decide against releasing version `1.0.0` together with this I'll change this :wink: